### DISCUSSION TITLE
Getting parent_id

### DISCUSSION THREAD

**Discussion by AndressMartin** [acheong08/ChatGPT#990] [Q&A] [answered]
Hello, so I am using the api for a discord bot implementation, and I have the following question: What is the best way to get the parent_id when you want to fetch and continue an old conversation?

When I call AsyncChatbot.ask(), I generate a new conversation with the name of the discord channel it was asked at. Then the parent_id is generated for the previous message. I am then saving the UUID of the conversation if anyone wants to pick it again in the future with get_conversations(). 

However, with parent_id I don't see the point of saving it somewhere since it changes. What is the best approach for getting the last message's parent_id?

**Comment by hrizal** [comment, marked as answer]
in response data you can get with
data["conversation_id"] and data["parent_id"]  
only work with same "access_token"

**Reply by AndressMartin** [reply]
Thanks!
